CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ about writing changes to this log.
 
 ## [Unreleased]
 
+## [1.1.1] 22.06.2023
```

Comment:
This is a slightly surprising heading for release 1.1.2 …

Reply:
Fixed.